Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM python:3.8-slim-bullseye
+FROM python:3.8-slim-bookworm
```

Comment:
@kanda999 @rinaldifirdaus Shall we also switch the Python version to 3.10 for 16.0, following suite to Doodba? In that case, we will have it tested in the staging server for a while as usual.